<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>北京</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/new/icon/images/bkImg.png</normaloff>:/new/icon/images/bkImg.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/new/icon/images/icon.png</normaloff>:/new/icon/images/icon.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广州</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/new/icon/images/bkImg.png</normaloff>:/new/icon/images/bkImg.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>深圳</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/new/icon/images/icon.png</normaloff>:/new/icon/images/icon.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addCityButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delCityButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityNameEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delAllButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAllButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示全部</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>294</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
